### DISCUSSION TITLE
ISAQB Module GREEN (18.11. - 19.11.)

### DISCUSSION THREAD

**Discussion by pkutschera** [Green-Software-Foundation/discussions#173] [Events]
On 18.11.2024 09:00 - 19.11.2024 17:00 we are hosting a two day course on Green IT (in German). The target audience are software developers, software architects as well as product owners with a technical background. 

At the end of the GREEN module, participants will know the key levers for reducing CO2 in IT. They will be able to assess the impact of software architecture and how architecture patterns and styles work. They know the pitfalls of data handling and can assess the impact of the selection of technical components. With regard to the cloud, participants know how to use the options offered there in terms of energy efficiency.

The course  topics are described in greater detail in the following document: https://www.isaqb.org/download/curriculum-module-green-green-software-development-of-resource-efficient-applications/?wpdmdl=19624.

**Comment by pkutschera** [comment]
Hi @NAMRATA-WOKE, what do I have to do to share the information about our course in the GSF newsletter?

**Reply by NAMRATA-WOKE** [reply]
@pkutschera is this a paid course?

**Reply by pkutschera** [reply]
Yes, it is.

**Comment by pkutschera** [comment]
@NAMRATA-WOKE What are now the next steps?
